src/utils/api.test.js

```diff
@@ -0,0 +1,119 @@
+import { fetchData } from './api';
+
+describe('fetchData', () => {
+  let path;
+  let method;
+  let data;
+
+  beforeEach(() => {
+    path = '/api/v1/projects';
+    method = 'POST';
+    data = {
+      id: "3",
+      type: "event",
+      attributes: {
+        name: "Flash Foxy Women's Climbing Festival",
+        city: "Canon Springs",
+        state: "CO",
+        event_type: "climbing",
+        price: 35,
+        start_date: "10-19-2018",
+        end_date: "10-21-2018",
+        description:
+          "This October 19-21, head on down to the best limestone sport cragging Colorado has to offer, and with the American Alpine Club and CAMP USA, and be treated to a weekend of climbing, community, clinics, contests, beer, slideshows, films, giveaways, stewardship and fun.",
+        event_url: "https://americanalpineclub.org/shelf-road-cc",
+        image_url:
+          "https://static1.squarespace.com/static/55830fd9e4b0ec758c892f81/57bc7284f7e0abe0efbb523e/59417dc21b631bad37b19a74/1497464437416/_MG_1604.jpg",
+        video_url: "https://www.youtube.com/embed/Cnjv_Brs2Bo"
+      }
+    };
+  });
```

Comment:
I'm wondering if here we could 'import {mockEvents}' from our mockData file and then just use the first index if we want to shorten this up! So instead it could be data = mockEvents.data[0].

Reply:
Good idea, I will update this.